src/server/send/sendAPI.js

```diff
@@ -72,10 +73,36 @@ const setup = (app: Router) => {
   app.post(
     '/send/recoveryinstructions',
     passport.authenticate('jwt', { session: false }),
-    onlyInEnv('production', 'staging', 'test', 'development'),
+    onlyInEnv('production', 'staging', 'test'),
     wrapAsync(async (req, res, next) => {
       const log = req.log.child({ from: 'sendAPI - /send/recoveryinstructions' })
       const { user } = req
+      const { mnemonic } = req.body
+      
+      log.info('sending recovery email', user)
+      //at this stage user record should contain all his details
+      await Mautic.sendRecoveryEmail(user, mnemonic)
+      res.json({ ok: 1 })
+    })
+  )
+  
+  /**
+   * @api {post} /send/magiclink Send recovery instructions email
+   * @apiName Recovery Instructions
+   * @apiGroup Send
+   *
+   * @apiParam {String} magicLine
+   *
+   * @apiSuccess {Number} ok
+   * @ignore
+   */
+  app.post(
+    '/send/magiclink',
+    passport.authenticate('jwt', { session: false }),
+    onlyInEnv('production', 'staging', 'test', 'development'),
+    wrapAsync(async (req, res, next) => {
+      const log = req.log.child({ from: 'sendAPI - /send/magiclink' })
+      const { user } = req
       const { magicLine } = req.body
       let userRec = user
       if (!user.mauticId || user.mauticId < 0) {
```

Comment:
we dont need this. user should have a mautic id by now. since he verified his email.

Reply:
[done](https://github.com/GoodDollar/GoodServer/pull/69/commits/6d201cb8da75faeb28c1696bbaa5975d18f82cff)